docker-compose.yml

```diff
@@ -38,7 +39,22 @@ services:
     volumes:
       - ./crontabs:/etc/crontabs:ro
     entrypoint: ["/usr/sbin/crond", "-f", "-l", "8"]
+  redis:
+    image: redis:5.0.8-alpine3.11
+    restart: unless-stopped
+    container_name: graphql-server-redis
+    volumes:
+      - "redisdata:/data"
+    networks:
+      - redis
+    ports:
+      - '6379:6379'
+    environment:
```

Comment:
Setting the environment turns out to be obsolete and therefore can be removed safely